src/modules/dashboard/lists/BorrowAssetsList/BorrowAssetsListItem.tsx

```diff
@@ -32,10 +31,6 @@ export const BorrowAssetsListItem = ({
   const { currentMarket } = useProtocolDataContext();
   const borrowButtonDisable = isFreezed || Number(availableBorrows) <= 0;
 
-  // Hide the asset to prevent it from being borrowed if borrow cap has been reached
-  const { borrowCap: borrowCapUsage, debtCeiling } = useAssetCaps();
```

Comment:
What will happen if an asset has reached the max borrow cap, will it still show up in the borrow positions and asset lists with a warning?

Reply:
Looks like it'll show up in the list with a warning/tooltip
![image](https://user-images.githubusercontent.com/23554636/213579740-e97dd051-7b41-48d5-9b33-e1f5b0a6d0ad.png)